HLTrigger/Configuration/python/HLT_75e33/paths/HLT_Ele30_WPTight_L1Seeded_LooseDeepTauPFTauHPS30_eta2p1_CrossL1_cfi.py

```diff
@@ -0,0 +1,610 @@
+from ..modules.hltEle32WPTightClusterShapeSigmawwL1SeededFilter_cfi import hltEle32WPTightClusterShapeSigmawwL1SeededFilter as _hltEle32WPTightClusterShapeSigmawwL1SeededFilter
+from ..modules.hltEle32WPTightHgcalHEL1SeededFilter_cfi import hltEle32WPTightHgcalHEL1SeededFilter as _hltEle32WPTightHgcalHEL1SeededFilter
+from ..modules.hltEgammaHoverEL1Seeded_cfi import *
+from ..modules.hltEle32WPTightHEL1SeededFilter_cfi import hltEle32WPTightHEL1SeededFilter as _hltEle32WPTightHEL1SeededFilter
+from ..modules.hltEgammaEcalPFClusterIsoL1Seeded_cfi import *
+from ..modules.hltEle32WPTightEcalIsoL1SeededFilter_cfi import hltEle32WPTightEcalIsoL1SeededFilter as _hltEle32WPTightEcalIsoL1SeededFilter
+from ..modules.hltEgammaHGCalLayerClusterIsoL1Seeded_cfi import *
+from ..modules.hltEle32WPTightHgcalIsoL1SeededFilter_cfi import hltEle32WPTightHgcalIsoL1SeededFilter as _hltEle32WPTightHgcalIsoL1SeededFilter
+from ..modules.hltEgammaHcalPFClusterIsoL1Seeded_cfi import *
+from ..modules.hltEle32WPTightHcalIsoL1SeededFilter_cfi import hltEle32WPTightHcalIsoL1SeededFilter as _hltEle32WPTightHcalIsoL1SeededFilter
+from ..modules.hltEle32WPTightPixelMatchL1SeededFilter_cfi import hltEle32WPTightPixelMatchL1SeededFilter as _hltEle32WPTightPixelMatchL1SeededFilter
+from ..modules.hltEle32WPTightPMS2L1SeededFilter_cfi import hltEle32WPTightPMS2L1SeededFilter as _hltEle32WPTightPMS2L1SeededFilter
+from ..modules.hltEle32WPTightGsfOneOEMinusOneOPL1SeededFilter_cfi import hltEle32WPTightGsfOneOEMinusOneOPL1SeededFilter as _hltEle32WPTightGsfOneOEMinusOneOPL1SeededFilter
+from ..modules.hltEle32WPTightGsfDetaL1SeededFilter_cfi import hltEle32WPTightGsfDetaL1SeededFilter as _hltEle32WPTightGsfDetaL1SeededFilter
+from ..modules.hltEle32WPTightGsfDphiL1SeededFilter_cfi import hltEle32WPTightGsfDphiL1SeededFilter as _hltEle32WPTightGsfDphiL1SeededFilter
+from ..modules.hltEle32WPTightBestGsfNLayerITL1SeededFilter_cfi import hltEle32WPTightBestGsfNLayerITL1SeededFilter as _hltEle32WPTightBestGsfNLayerITL1SeededFilter
+from ..modules.hltEle32WPTightBestGsfChi2L1SeededFilter_cfi import hltEle32WPTightBestGsfChi2L1SeededFilter as _hltEle32WPTightBestGsfChi2L1SeededFilter
+from ..modules.hltEgammaEleL1TrkIsoL1Seeded_cfi import *
+from ..modules.hltEle32WPTightGsfTrackIsoFromL1TracksL1SeededFilter_cfi import hltEle32WPTightGsfTrackIsoFromL1TracksL1SeededFilter as _hltEle32WPTightGsfTrackIsoFromL1TracksL1SeededFilter
+from ..modules.hltEgammaEleGsfTrackIsoV6L1Seeded_cfi import *
+from ..modules.hltEle32WPTightGsfTrackIsoL1SeededFilter_cfi import hltEle32WPTightGsfTrackIsoL1SeededFilter as _hltEle32WPTightGsfTrackIsoL1SeededFilter
+from ..modules.hltAK4PFJetsForTaus_cfi import *
+from ..modules.hltHpsSelectedPFTauLooseTauWPDeepTau_cfi import * 
+from ..modules.hltHpsPFTau30LooseTauWPDeepTau_cfi import * ## NEW (with threshold different from youngwan)
+from ..modules.hltAK4PFJets_cfi import *
```

Comment:
I think the module imported from this line is never used. I'd suggest to verify if that's correct and, if so, remove the import.

Reply:
Done